### PR TITLE
Apply php8.1 in Rector preset

### PR DESCRIPTION
**Pull request by samsonasik**
| Q             | A
| ------------- | ---
| QA?  | ✔️

Continue apply php 8.1 set.

### CODE REVIEW

**Comment by samsonasik** [comment]
Ready to merge 👍

File: src/Queue/src/Interceptor/Consume/Handler.php [hunk 1]
```diff
@@ -19,14 +19,12 @@
  */
 final class Handler
 {
-    private readonly TracerFactoryInterface $tracerFactory;
     private readonly bool $isLegacy;
 
     public function __construct(
         private readonly HandlerInterface|CoreInterface $core,
-        ?TracerFactoryInterface $tracerFactory = null,
+        private readonly TracerFactoryInterface $tracerFactory = new NullTracerFactory(new Container()),
```
Comment (samsonasik) [inline src/Queue/src/Interceptor/Consume/Handler.php:26]: @roxblnfk do you think it is bc break, if yes, I can skip `NewInInitializerRector`

Comment (roxblnfk) [reply src/Queue/src/Interceptor/Consume/Handler.php:26]: I think it's better to disable it here and apply it for v4 in the future.

Comment (samsonasik) [reply src/Queue/src/Interceptor/Consume/Handler.php]: sure, I revered and skip `NewInInitializerRector` https://github.com/spiral/framework/pull/1141/commits/9cf5a48640591b4f997008b02789ac8c59ed15bd 👍

**Comment by samsonasik** [comment]
Ready to merge 👍

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**